src/Vehicle/Vehicle.cc

```diff
@@ -4002,6 +4010,53 @@ void Vehicle::gimbalControlValue(double pitch, double yaw)
         MAV_MOUNT_MODE_MAVLINK_TARGETING);   // MAVLink Roll,Pitch,Yaw
 }
 
+void Vehicle::gimbalPitchStep(int direction)
+{
+    if(_haveGimbalData) {
+        //qDebug() << "Pitch:" << _curGimbalPitch << direction << (_curGimbalPitch + direction);
+        double p = static_cast<double>(_curGimbalPitch + direction);
+        gimbalControlValue(p, static_cast<double>(_curGinmbalYaw));
```

Comment:
curGinmbalYaw shall be renamed to curGimbalYaw

Reply:
Yup! :P